config/jobs/image-pushing/releng/k8s-staging-artifact-promoter.yaml

```diff
@@ -3,7 +3,7 @@ postsubmits:
     - name: post-kpromo-push-image
       cluster: k8s-infra-prow-build-trusted
       annotations:
-        testgrid-dashboards: sig-release-releng-blocking, sig-release-master-informing, sig-release-image-pushes, sig-k8s-infra-gcb
+        testgrid-dashboards: sig-release-releng-blocking, sig-release-image-pushes, sig-k8s-infra-gcb
         testgrid-alert-email: k8s-infra-alerts@kubernetes.io, release-managers+alerts@kubernetes.io, release-team@kubernetes.io
```

Comment:
TODO: probably doesn't need to alert release team? (Good question for the release lead perhaps?)

Reply:
I'd be inclined towards removing this from release team's radar. This falls on the releng side, so it's fine to remove release team from observing these.